src/actions/receive-profile.js

```diff
@@ -820,12 +808,16 @@ export function retrieveProfileFromFile(
             dispatch(receiveZipFile(zip));
           }
           break;
-        default:
-          dispatch(
-            errorReceivingProfileFromFile(
-              new Error(`Unable to load a file of type "${file.type}"`)
-            )
-          );
+        default: {
+          // No type? Let's parse it as a JSON serialized profile.
```

Comment:
Can you document that you are assuming `'application/json'` here?

Reply:
Actually that's the point of this patch: we don't assume anything anymore, especially no mime type. We try to parse as json as a last resort.
Isn't the existing comment good enough ?

Reply:
I find the comment confusing as it states that it has no type, while in fact it could be 3 things:

 * `application/json` - The correct mime type that needs to be processed.
 * `image/jpeg` et al. - Some arbitrary unsupported mime type, that we may explicitly not support.
 * blank - No mime type supported.

In each of these cases this code branch is trying to load it as JSON.

Reply:
I see what you mean, I'll update the comment in something better

Reply:
I used:
```
          // Plain uncompressed profile files can have file names with uncommon
          // extensions (eg .profile). So we can't rely on the mime type to
          // decide how to handle them. We'll try to parse them as a plain JSON
          // file.
```